torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -590,3 +598,65 @@ def encode_forward(self, feat: Tensor) -> Tensor:
         # -> [batch_size, num_cols, out_channels]
         x = x_lin + self.bias
         return x
+
+
+class MultivariateLSTMEncoder(StypeEncoder):
+    r"""An LSTM for numerical features. The columns represent different
+    input variables and each row represent a timestamp. The encoder
+    generates one embedding of size `out_channels` for the entire
+    multi-variate timeseries. The structure of LSTM is described in
+    `"Long Short-Term Memory"
+    <https://dl.acm.org/doi/10.1162/neco.1997.9.8.1735>`_ paper.
+
+    Args:
+        hidden_channels (int): Hidden channels of LSTM.
+            (default: 16)
+        num_layers (int): Number of layers in LSTM.
+            (default: 2)
+
+    Returns:
+        torch.Tensor: The output embeddings of size
+            [1, out_channels].
+    """
+    supported_stypes = {stype.numerical}
```

Comment:
Sorry for my confusion: How is this useful for `stype.numerical`? Isn't this something we would want to add for `sequential`?